core/core_bind.h

```diff
@@ -431,9 +431,19 @@ class ClassDB : public Object {
 	static void _bind_methods();
 
 public:
+	enum APIType {
+		API_CORE,
+		API_EDITOR,
+		API_EXTENSION,
+		API_EDITOR_EXTENSION,
+		API_NONE,
+	};
+	APIType get_api_type(const StringName &p_class) const;
 	PackedStringArray get_class_list() const;
 	PackedStringArray get_inheriters_from_class(const StringName &p_class) const;
 	StringName get_parent_class(const StringName &p_class) const;
+	StringName get_property_setter(const StringName &p_class, const StringName &p_property) const;
+	StringName get_property_getter(const StringName &p_class, const StringName &p_property) const;
```

Comment:
Should the word `class` be in the method name somewhere?

I feel like all the other methods that are getting something about a particular class have it added, like:

- `get_integer_constant_enum` -> `class_get_integer_constant_enum`
- `is_enum_bitfield` -> `is_class_enum_bitfield`

Reply:
Thanks, this looks good to me!